File: src/main/java/com/senderman/telecrafter/telegram/TelegramChat.java
package com.senderman.telecrafter.telegram;

import com.google.inject.Inject;
import com.senderman.telecrafter.Config;

public class TelegramChat {

    private final TelecrafterBot bot;
    private final Config config;

    @Inject
    public TelegramChat(TelecrafterBot bot, Config config) {
        this.bot = bot;
        this.config = config;
    }

    public void sendMessage(String message) {
        bot.sendMessageAsync(config.getChatId(), message);
    }

}

File: src/main/kotlin/com/senderman/telecrafter/telegram/TelegramChat.kt
package com.senderman.telecrafter.telegram

import com.google.inject.Inject
import com.senderman.telecrafter.Config
import org.telegram.telegrambots.meta.api.methods.send.SendMessage

class TelegramChat @Inject constructor(
    config: Config,
    private val bot: TelecrafterBot
) {
    private val chatId: Long = config.chatId

    fun sendMessage(message: String) {
        bot.execute(SendMessage(chatId, message).enableHtml(true))
    }
}